frontend/tests/features/crudCollection.test.ts

```diff
@@ -0,0 +1,42 @@
+import { TEST_ENV, TEST_URL } from "tests/common/constants";
+import { goToPage } from "tests/utils/helpers";
+import { getText } from "tests/utils/selectors";
+
+const describeIfCalledByDevEnv = TEST_ENV === "dev" ? describe : describe.skip;
+
+describeIfCalledByDevEnv("CRUD Collection", async () => {
+  it("creates and deletes a collection", async () => {
+    await goToPage(TEST_URL);
+    // LOG IN
+    await page.click(getText("Log In"));
+    await page.waitForNavigation({ url: TEST_URL, waitUntil: "networkidle" });
+    // Create collection
+    await page.click(getText("Create Collection"));
+    await page.fill("#name", "TEST COLLECTION");
+    await page.fill("#description", "TEST DESCRIPTION");
+    await page.fill("#contact-name", "TEST");
+    await page.fill("#contact-email", "TEST@example.coms");
+    await page.click("input[type=checkbox]");
+    const [request, response] = await Promise.all([
+      page.waitForEvent("request"),
+      page.waitForEvent("response"),
+      await page.click(getText("Create")),
+    ]);
+
+    const cookie = request.headers().cookie;
+
+    const collectionID = (await response.json()) as object;
+
+    await expect(page).toHaveSelector("h3");
```

Comment:
For robustness and readability, let's also add `test-id` attribute for the element we're checking here too. `h3` doesn't convey the meaning and intention of why checking that element is necessary in this test. Also whatever that element is, if the design decides to change it to `h4` or `h3`, this test will break 🙏